src/components/ErrorPage.tsx

```diff
@@ -40,6 +40,25 @@ ${error?.stack ?? "No stack trace"}
   useEffect(updateHeight, []);
   useEventListener("resize", updateHeight);
 
+  let modifiedError: Error | null = null;
+  if (error instanceof Error) {
+    // handle dynamic module loading error
+    if (
+      error.message
+        .toLowerCase()
+        .includes("failed to fetch dynamically imported module")
+    ) {
+      // don't mutate the original error
+      modifiedError = new Error(
+        `${error.message}. This is likely due to a network issue or the lxd server being unreachable.`,
+      );
+      modifiedError.stack = error.stack;
+      modifiedError.name = error.name;
+    }
+  }
+
+  const displayError = modifiedError || error;
+
```

Comment:
Do we really need to modify the error? I think we can modify the rendering of the error message below and append the text there, which seems much simpler and more straight forward.